pkg/hive/cell/structured.go

```diff
@@ -568,15 +571,23 @@ type subReporter struct {
 
 func (s *subReporter) OK(message string) {
 	if err := s.base.setStatus(s.id, StatusOK, message, nil); err != nil {
-		log.WithError(err).Warnf("could not set OK status on subreporter %q", s.id)
+		if errors.Is(err, errReporterStopped) {
+			log.WithError(err).Debugf("could not set OK status on subreporter %q", s.id)
+		} else {
+			log.WithError(err).Warningf("could not set OK status on subreporter %q", s.id)
+		}
```

Comment:
Sounds good, there doesn't appear to be a general logfields.ID field, so I just added a local const here since this is only for logging internal details regarding health.